hazelcast/src/main/java/com/hazelcast/sql/impl/expression/RowValue.java

```diff
@@ -32,6 +33,7 @@
  * field names - names are part of the type, the value doesn't reference the
  * type.
  */
+@Beta
```

Comment:
We don't need it here because it is and will remain private API. Maybe we should mention it explicitly in the javadoc.

Reply:
Its actually supposed to become public API similar to HazelcastJsonValue, its just a question of what it should look like.